ProcessMaker/modeler
# Observation/FOUR-12816: Error on Modeler describing OpenAI not being available

## Issue & Reproduction Steps

The following error message appears when Modeler is initialized:

The POST method is not supported for route api/1.0/package-ai/getPromptSessionHistory. Supported methods: GET, HEAD.

## Solution
- When promptsessionid does not exists call again to generate a new one and hide the error message to the user

## How to Test 
- Go to modeler
- Inspect the network and get the promptsessionid sent to the getPromptSessionHistory endpoint
- Go to the database and modify that promptSessionId in the table ai_prompt_session of the microservice database
- Reload modeler (should fail and show the error message)

After apply the fix themessage should not be show and the endpoint should work correctly.

## Related Tickets & Packages
- [FOUR-12816](https://processmaker.atlassian.net/browse/FOUR-12816)

## Code Review Checklist


[FOUR-12816]: https://processmaker.atlassian.net/browse/FOUR-12816?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

.
